revision-static-assets/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "revision-static-assets",
+  "version": "1.0.0",
+  "description": "Give files a unique filename based on their contents and update references to them.",
+  "scripts": {
+    "prebuild": "rimraf dist/",
+    "build": "ncp src/ dist/",
+    "postbuild": "npm run revision",
+    "revision": "node scripts/revision.js",
+    "start": "http-server './dist/' -c-1 -o -p 3131"
```

Comment:
set caching to what you suggest

Reply:
I can change this to `http-server './dist/' -c31536000 -o -p 3131` but that also sets `max-age=31536000` to non-revisioned files like the HTML. What's better? I feel like adding a native Node HTTP or connect setup is too much.